index.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+$db = new PDO(
+    'mysql:host=192.168.20.20; dbname=Project2',
+    'root',
+    '');
+
+$db->setAttribute(
+    PDO::ATTR_DEFAULT_FETCH_MODE,
+    PDO::FETCH_ASSOC);
+
+$sql = 'SELECT * FROM `Plants`';
+$query = $db->query($sql);
```

Comment:
We use query method on $db, we don't need to know what query() returns. since you know what it will return-an associative array-, I'm not sure if it's the best to use query method here. 
Suggestion: use prepare() method instead maybe?